Task 5/middlewares/JWTVerificationMiddleware.js

```diff
@@ -0,0 +1,17 @@
+import jwt from 'jsonwebtoken';
```

Comment:
No need to add "Middleware" making every function name non-readable.
Consider removing it.